react/PlaceDetails.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react'
+import { Dropdown } from 'vtex.styleguide'
+
+const address: { [index: string]: any } = {
+  street: 'Av. Belgrano',
+  number: 2248,
+  complement: '',
+  postalCode: '2000',
+  city: 'Rosario',
+  state: 'Santa Fe',
+}
+
+const countryDescriptions: CountryDescription[] = [
+  {
+    name: 'ARG',
+    summary: [
+      [{ name: 'street' }, { delimiter: ' ', name: 'number' }],
+      [{ name: 'complement' }],
+      [{ name: 'postalCode' }],
+      [{ name: 'city' }, { delimiter: ', ', name: 'state' }],
+    ],
+  },
+  {
+    name: 'BRA',
+    summary: [
+      [
+        { name: 'street' },
+        { delimiter: ' ', name: 'number' },
+        { delimiter: ', ', name: 'complement' },
+      ],
+      [
+        { name: 'neighborhood', delimiterAfter: ' - ' },
+        { name: 'city' },
+        { delimiter: ' - ', name: 'state' },
+      ],
+      [{ name: 'postalCode' }],
+    ],
+  },
+  {
+    name: 'KOR',
+    summary: [
+      [{ name: 'street' }, { delimiter: ', ', name: 'complement' }],
+      [
+        { name: 'city' },
+        { delimiter: ', ', name: 'state' },
+        { delimiter: ' ', name: 'postalCode' },
+      ],
+    ],
+  },
+]
```

Comment:
I would move all this mock data to a separate file.

Reply:
Done.
Is the new file name (and location) correct? Do you guys have any convention on this?

Reply:
In the future that file will almost certainly disappear (and I'd use real data instead), but it will be useful to know for future occasions.

Reply:
You can put the mock data in a file in the `react/__mocks__` folder, so in the future you can use the same data to write unit tests ; )

Reply:
Awesome, I'll do that!